eng/Workarounds.targets

```diff
@@ -21,10 +21,15 @@
   </ItemGroup>
 
   <!-- Workaround https://github.com/aspnet/AspNetCore/issues/7503. This chains GenerateSourceLinkFile before razor component targets run. -->
-  <Target Name="_EnsureSourceLinkHappensBeforeRazorComponentGeneration"
+  <!-- Workaround https://github.com/dotnet/source-build/issues/1112. Source link is currently disabled in source build so do not apply this worksaround. -->
```

Comment:
Should either restore the link to our tracking bug (https://github.com/aspnet/AspNetCore/issues/7503) or include it too.

Also, restore comments describing what the target does.

Reply:
I didn't remove anything?

Reply:
Not sure why but I thought line 23 was removed.

:ok: just make it clear #7503 tracks dotnet/source-build#1112

Reply:
I don't think it's quite the same. The issue described in #7503 is general to source link. dotnet/source-build#1112 is specific to source build.